go.mod

```diff
@@ -4,6 +4,7 @@ require (
 	github.com/cenkalti/backoff v2.2.1+incompatible
 	github.com/fatih/structs v1.1.0
 	github.com/fsnotify/fsnotify v1.4.9
+	github.com/ghodss/yaml v1.0.0
```

Comment:
https://github.com/go-yaml/yaml is much more popular and also used by viper

Reply:
right I used the same as in kratos